numerapi/numerapi.py

```diff
@@ -105,7 +105,7 @@ def download_current_dataset(self, dest_path=".", dest_filename=None,
 
         if os.path.exists(dataset_path):
             self.logger.info("target file already exists")
-            return dataset_path
+            # return dataset_path
```

Comment:
just delete that line, no need to leave that commented

Reply:
and if you move that log message to utils.py, that complete if statement can go

Reply:
Got it, I'll remove the log from here and add it in utils.py. There are a total of 4 cases and I'll log for all of them : 

```python
    if file_exists:
        logger.info("target file already exists")
        if download_incomplete:
            # Download complete
            logger.info("resuming download")
        elif download_complete:
            # Download complete
            logger.info("download complete")
            return
        else:
            # Error, delete file and restart download
            logger.error("deleting file and restarting")
    else:
        # File does not exist, starting download
        logger.info("starting download")
```